### PR TITLE
AMP live blog byline contributor image

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Byline images in live blogs always use `` tags, even in AMP live blogs. Vanilla `` tags are not support in AMP articles, so the articles were not validating. This change introduces `<amp-img>` for byline contributor images in AMP live blogs.
## Does this affect other platforms - Amp, Apps, etc?

Nope

### CODE REVIEW

File: common/app/views/fragments/amp/stylesheets/liveBlog.scala.html [hunk 1]
```diff
@@ -78,11 +78,11 @@
...
 
 .liveblog-block-byline__img {
-    display: inline;
-    border-radius: 50%;
-    max-width: 2.25rem;
     vertical-align: middle;
-    margin-right: 0.25rem;
+}
+
+.liveblog-block-byline__img img {
```
Comment (gtrufitt) [inline common/app/views/fragments/amp/stylesheets/liveBlog.scala.html]: Guessing this might be an amp thing, but should we be styling elements directly?

**Comment by gtrufitt** [comment]
Minor comment but otherwise 👍

File: common/app/views/fragments/amp/stylesheets/liveBlog.scala.html (hunk 1 above)
Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/liveBlog.scala.html]: Ideally we shouldn't be... We should probably stick to styling `amp-img` instead.

Comment (SiAdcock) [inline common/app/views/fragments/amp/stylesheets/liveBlog.scala.html]: I could not find a way to style this element directly without nesting. We could always dispense with the rounded borders for the contributor image in AMP, I guess. The only classes we can hook into are generic classes provided by AMP.

Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/liveBlog.scala.html]: Why not just style `liveblog-block-byline__img`? You can put the border-radius on that.

Comment (NataliaLKB) [inline common/app/views/fragments/amp/stylesheets/liveBlog.scala.html]: If you do that, you also lose a couple new lines of code 😉

**Comment by SiAdcock** [comment]
Thanks all, I've applied the border radius to `amp-img` and it's all working. Diagnosis: the border radius wasn't being applied when emulating devices in Chrome's device emulator. You have to disable the emulator, reload the page, then re-enable the emulator to pick up the border radius 😖 

I've tested it on Chrome and Safari for iOS and it works fine in real life.

**Comment by NataliaLKB** [comment]
Wow! Well done for figuring it out.

**Pull request merged**